WcaOnRails/app/models/competition.rb

```diff
@@ -747,15 +747,15 @@ def managers
 
   def receiving_registration_emails?(user_id)
     competition_delegate = competition_delegates.find_by_delegate_id(user_id)
-    if competition_delegate&.receive_registration_emails
+    if !competition_delegate&.receive_registration_emails
```

Comment:
Why are you negating the condition here?
The purpose of this method is to check whether a user should receive registration emails. Your updated check is basically saying "If the flag `receive_registration_emails` is NOT set, reply that they should receive emails" which sounds very wrong 🙈